src/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -316,6 +316,37 @@ internal static unsafe void FixupMethodCell(IntPtr hModule, MethodFixupCell* pCe
             }
         }
 
+#if !PLATFORM_UNIX
+        private static unsafe IntPtr GetProcAddress(IntPtr hModule, byte* methodName, CharSet charSetMangling)
+        {
+            var exactMatch = Interop.mincore.GetProcAddress(hModule, methodName);
+
+            if ((charSetMangling == CharSet.Ansi && exactMatch != IntPtr.Zero) || charSetMangling == 0)
+            {
+                return exactMatch;
+            }
+
+            int nameLength = strlen(methodName);
+
+            byte* probedMethodName = stackalloc byte[nameLength + 1];
```

Comment:
I assume the `+ 1` is for the suffix but we need another byte for the null terminator.

Reply:
If you can use a span here, that would be a good way to avoid a possible stack buffer overrun.